pages/instructorCourseAdminStaff/instructorCourseAdminStaff.ejs

```diff
@@ -218,6 +226,11 @@
           </tbody>
         </table>
         <small class="card-footer">
+          <% if (hasUnknownUsers) { %>
+          <p>
+            Users with name "<span class="text-danger">Unknown user</span>" either have never logged in or have an incorrect UID.
+          </p>
```

Comment:
My only concern is that the comment at the bottom can easily be missed. Would a bootstrap `alert` box be too much?
```suggestion
          <p class="alert alert-danger">
            Users with name "<span class="text-danger">Unknown user</span>" either have never logged in or have an incorrect UID.
          </p>
```

Reply:
Good thinking. I made it an `alert-warning`.